docs/intro/overview.md

```diff
@@ -0,0 +1,56 @@
+---
+sidebarLabel: Overview
+title: "What is Solana?"
+sidebarSortOrder: 1
+description:
+  "Solana is the high-performance blockchain designed for mass adoption. Learn
+  why Solana is the top choice for developers looking to build scalable
+  blockchain applications."
+altRoutes:
+  - /docs/intro/history
+  - /docs/intro
+---
+
+Solana is a blockchain built for mass adoption. It's a high performance network
+that is utilized for a range of use cases, including finance, payments, and
+gaming. Solana operates as a single global state machine, and is open,
+interoperable, and decentralized.
```

Comment:
```suggestion
Solana is a high performance blockchain used for finance, payments, gaming and more. Solana operates as a single global state machine, and is open,
interoperable, and decentralized.
```

Reply:
This is actually from our internal boilerplate. It's how we're supposed to write about Solana 🤷